src/watch/fsevents-importer.ts

```diff
@@ -1,8 +1,10 @@
-let fsEvents: typeof import('fsevents');
+let fsEvents: any; // TODO find better type solution, fsevents is only being installed on macOS
```

Comment:
As the actual type does not really matter, I would use `unknown` instead of `any` in both places.

Reply:
I initially used `unknown`, I thought I run into some issue with it.